scripts/run_generative.py

```diff
@@ -140,6 +140,8 @@ def main():
         is_prometheus = True
     elif "gemini" in args.model:
         model_modifier = "gemini"
+    elif "Llama-3.1" in args.model:
```

Comment:
Is this the complete diff? How is the model modifier being handled? 🤔

Reply:
It was already there in main: https://github.com/for-ai/aya_rm_multilingual/blob/41a8292ba70152fe5d22ad9823ddf4997767b113/scripts/run_generative.py#L278